app/models/setting/provisioning.rb

```diff
@@ -29,10 +29,10 @@ def self.local_boot_labels
     'lo',
     'en*v*',
     'usb*',
-    'vnet*',
+    '*vnet*',
```

Comment:
Although Mellanox driver does use vnet suffix (https://github.com/torvalds/linux/blob/f40ddce8/drivers/net/ethernet/mellanox/mlx5/core/dev.c#L208) I do not think this was the reason why we added this.

The reason was QEMU. When using Linux virtualization via libvirt, then bridge interfaces, the offenders we want to filter out, are named `virbr0`. However when using QEMU directly typically `tap` interface is used with naming convention of `vnet0`.